backend/query/src/profiles/models/profile.model.ts

```diff
@@ -0,0 +1,76 @@
+import { ApiResponseProperty } from '@nestjs/swagger';
+import { prop } from '@typegoose/typegoose';
+import { Expose, Type } from 'class-transformer';
+
+export class ProfileCounts {
+  @Expose()
+  @prop()
+  @ApiResponseProperty()
```

Comment:
I know it's annoying, but please add descriptions for the props. Especially for the frontend this is needed to label them correctly.
This does require you to use @ApiProperty()

Example:

tagsUsedOnCreation => Is this unique tags or just a general sum